src/test/kotlin/no/nav/familie/tilbake/config/OAuth2AccessTokenTestConfig.java

```diff
@@ -0,0 +1,28 @@
+package no.nav.familie.tilbake.config;
+
+import no.nav.security.token.support.client.core.oauth2.OAuth2AccessTokenResponse;
+import no.nav.security.token.support.client.core.oauth2.OAuth2AccessTokenService;
+import org.mockito.Mockito;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@Configuration
+@Profile("mock-oauth")
```

Comment:
Hvorfor trenger vi den?

Reply:
For å kunne kjøre integrasjonstester mot rest-endepunkter.